build.sbt

```diff
@@ -305,8 +312,17 @@ lazy val nativeServer: Project = (project in file("native-server"))
     dockerUsername := Some("softwaremill"),
     packageBin in GraalVMNativeImage := (packageBin in GraalVMNativeImage)
       .dependsOn(yarnTask.toTask(" build"))
-      .dependsOn(createBuildx)
       .value,
+    publish in Docker := {
```

Comment:
aren't the above the same? If so, we should extract them to a settings list like `dockerBuildxSettings`

Reply:
Ok, extracted.